response_operations_ui/templates/account/activate-account.html

```diff
@@ -88,13 +88,10 @@ <h1>Activate your account</h1>
                         "id": "password",
                         "name": "password",
                         "label": {
-                            "text": 'Password'
-                        },
-                        "attributes": {
-                            "autocomplete": 'off'
+                            "text": "Password"
                         },
                         "showPasswordText": "Show password",
-                        "error": passwordError
+                        "error": {"text": errorData[0]["text"] } if errorData else None
```

Comment:
This isn't ideal and I will raise a card as this page has a number of issues. At present everything gets returned via form.password.errors and because of that there will never be an error from form.password_confirm.errors which means there is redundancy in this file. Because nothing is attributed to confirm password having an error field on the component is also never going to work and why I removed it. 
For the moment the first password field will deal with errors and show the first elements text